central/graphql/resolvers/resolver.go

```diff
@@ -209,6 +212,10 @@ func New() *Resolver {
 		resolver.ImageCVEEdgeDataStore = imageCVEEdgeDataStore.Singleton()
 		resolver.ImageCVEDataStore = imageCVEDataStore.Singleton()
 	}
+	if features.FlattenImageData.Enabled() {
```

Comment:
2 things:
1. I would recommend a comment here as to what you are doing.
2. I would pull `ImageDataStore: imageDatastore.Singleton(),` from the giant assignment above and do it in the else.  The reason being is it will read better AND will be easier to remove when we clean up.